codes/quantum/qudits/reflexive_stabilizer.yml

```diff
@@ -0,0 +1,49 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+code_id: reflexive_stabilizer
+physical: qudits
+logical: qudits
+
+name: 'Vandermolen-Wright reflexive stabilizer code'
+introduced: '\cite{arxiv:2110.08414}'
+
+description: 'Let \(d\) be a power of a prime (i.e., \(d = p^m \) for some prime \(p\) and an integer \(m\)). Then, any \(a \in [d]\)
+could be expressed as a string of length \(m\) constructed from elements in \(\{1,2,\cdots,p\}\). Denote such a representation as \( (\alpha_1,\cdots,\alpha_m)\), with each \( \apha_i \in [p-1]\).
+Let us now define the discrete Weyl operators \( X(a)\) and \(Z(a)\) for each \( a\in [d]\)
+\begin{align}

```

Comment:
These Pauli matrices should be related to Galois-qudit Pauli matrices, which will be defined in galois_stabilizer (I've added where in that entry). However, these specific Paulis avoid use of the "trace" operation and instead use inner product (Remark 2.1.2 of paper). If these inner-product Paulis are identical to the Galois-qudit Paulis, then this equation is not needed (I suspect they are). Otherwise, you can keep this equation and clarify that these Paulis are different from the Galois-qudit ones.
<img width="689" alt="image" src="https://user-images.githubusercontent.com/8660306/156237675-de010acf-2702-4b2a-baac-577e8e1cd7c4.png">